neurolib/models/model.py

```diff
@@ -335,6 +323,8 @@ def storeOutputsAndStates(self, t, variables, append=False):
 
     def setInitialValuesToLastState(self):
         """Reads the last state of the model and sets the initial conditions to that state for continuing a simulation."""
+        if not hasattr(self, "t"):
+            raise ValueError("You tried using continue_run=True on the first run.")
```

Comment:
Why do we have to error here? The user could user `continue_run=True` in the first call as well.

Reply:
addressed this in the latest commit